tests/models/test_scarches.py

```diff
@@ -163,7 +163,7 @@ def test_scanvi_online_update(save_path):
     new_labels = adata1.obs.labels.to_numpy()
     new_labels[0] = "Unknown"
     adata1.obs["labels"] = pd.Categorical(new_labels)
-    setup_anndata(adata1, batch_key="batch", labels_key="labels")
+    _setup_anndata(adata1, batch_key="batch", labels_key="labels")
```

Comment:
should this be SCANVI.setup_anndata (same with below)

Reply:
resolving as dupe of the one on another test